python/ray/_private/runtime_env/packaging.py

```diff
@@ -956,12 +956,16 @@ async def install_wheel_package(
         # TODO(architkulkarni): Use `await check_output_cmd` or similar.
         exit_code, output = exec_cmd_stream_to_logger(pip_install_cmd, logger)
     finally:
-        if Path(wheel_uri).exists():
-            Path(wheel_uri).unlink()
+        wheel_uri_path = Path(wheel_uri)
```

Comment:
this was actually a minor bug... `wheel_uri` can be a directory in the case of `py_modules`, and `unlink` can only be used on regular files